### PR TITLE
FEATURE: Add slider component

### PR DESCRIPTION
**Pull request by markusguenther**

### CODE REVIEW

**Comment by markusguenther** [comment]
Bildschirmfoto 2019-11-20 um 16 08 07

Current state.

**Comment by markusguenther** [comment]
@itscursedphil @johannessteu

Hopefully ready for review :)

File: bundlesize.config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "files": [
     {
       "path": "./packages/components/dist/*.js",
-      "maxSize": "120 kB"
+      "maxSize": "125 kB"
```
Comment (johannessteu) [inline bundlesize.config.json]:
```suggestion
      "maxSize": "130 kB"
```

Comment (johannessteu) [reply bundlesize.config.json]: Let's just give it some more "space"

File: packages/components/src/Slider/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  name: string;
+  value: any;
+  onChange?: (value: number) => void;
+}
+
+const StyledSlider = styled.div<SliderProps>`
+  position: relative;
+  display: flex;
+  height: auto;
+  margin-left: ${`${theme.space[2]}px`};
```
Comment (johannessteu) [inline packages/components/src/Slider/Slider.tsx]: As we spread the `...marginProps` on it we should use the margin function here, right?
```suggestion
  ${margin}
```

**Pull request merged**